pandas/core/strings.py

```diff
@@ -898,23 +898,76 @@ def str_join(arr, sep):
 
 def str_findall(arr, pat, flags=0):
     """
-    Find all occurrences of pattern or regular expression in the
-    Series/Index. Equivalent to :func:`re.findall`.
+    Find all occurrences of pattern or regular expression in the Series/Index.
+
+    Equivalent to apply :func:`re.findall` to all the elements in the
+    Series/Index.
 
     Parameters
     ----------
-    pat : string
-        Pattern or regular expression
-    flags : int, default 0 (no flags)
-        re module flags, e.g. re.IGNORECASE
+    pat : str
+        Pattern or regular expression.
+    flags : int
+        re module flags, e.g. re.IGNORECASE (default is 0, which means
+        no flags).
+    kwargs
+        These parameters will be ignored.
 
     Returns
     -------
-    matches : Series/Index of lists
+    Series/Index of lists of strings
+        All non-overlapping matches of pattern or regular expression in each
+        string of this Series/Index.
 
     See Also
     --------
-    extractall : returns DataFrame with one column per capture group
+    extractall : For each subject string in the Series, extract groups \
+    from all matches of regular expression pat.
```

Comment:
Can you indent this (one 4-space indentation)? To make clear it belongs to the link above (the same for re.findall below)